coffee/offline.coffee

```diff
@@ -181,7 +186,7 @@ Offline.checks.image = ->
   img = document.createElement 'img'
   img.onerror = Offline.markDown
   img.onload = Offline.markUp
-  img.src = Offline.getOption('checks.image.url')
+  img.src = Offline.getOption('checks.image.url') + cacheBust()
```

Comment:
> what if this url already has query params?

Good question, I could add detection of the query parameters and just tack on a non-sense parameter.  Would that be good enough?